package.json

```diff
@@ -10,11 +10,14 @@
   ],
   "scripts": {
     "lint": "eslint src",
-    "build:lib": "babel src --out-dir lib",
+    "build:lib": "babel src --out-dir lib && flow-copy-source -v src lib",
     "build:umd": "webpack src/index.js dist/react-flip-move.js --config webpack.config.production.js",
     "build:umd:min": "webpack src/index.js dist/react-flip-move.min.js --config webpack.config.production.min.js",
```

Comment:
Is anything needed for the UMD builds? I imagine we'd still have to remove all the types somehow.